frontend/test/resolution/testPromotion.cpp

```diff
@@ -101,7 +101,7 @@ static void runProgram(std::vector<const char*> prog, F&& f, Types... types) {
   auto context = &ctx;
   ErrorGuard guard(context);
 
-  std::string program = "module ChapelBase {\n  enum iterKind { standalone, leader, follower };\n}\n";
+  std::string program = "module ChapelBase {\n  enum iterKind { standalone, leader, follower };\noperator =(ref lhs: int, const rhs: int) {}\noperator =(ref lhs:real, const rhs:real) {}\n}\n";
```

Comment:
make this line less long? You could use `+` or just put the string literals next to each other (for preprocessor glomming) or you could use the C++ "string until custom terminator" facility.